### DISCUSSION TITLE
Date format and manually inputting time

### DISCUSSION THREAD

**Discussion by DAL70** [CubeLabsNZ/CubeTime#160] [Q&A]
First question is, could you change the date format to month/day/year?

Second question: I solved a puzzle on the wrong cube (4x4 and was actually 5x5), and it will not let me transfer it to another cube. Is there a way to manually input a time?

**Comment by pdtxie** [comment]
We’re planning to add localisation soon, so you’ll be able to change the date format, etc.

Yes, you can manually input a time by swiping down and pressing the “+”.
